flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -364,6 +364,10 @@ under the License.
 					<groupId>org.apache.calcite.avatica</groupId>
 					<artifactId>avatica</artifactId>
 				</exclusion>
+				<exclusion>
```

Comment:
can you add these exclusions to pom of flink-sql-client too to keep them in sync?